src/google/protobuf/repeated_field.h

```diff
@@ -1244,8 +1244,8 @@ PROTOBUF_NOINLINE void RepeatedField<Element>::GrowNoAnnotate(bool was_soo,
   if (old_size > 0) {
     Element* pnew = static_cast<Element*>(new_rep->elements());
     Element* pold = elements(was_soo);
-    // TODO: add absl::is_trivially_relocatable<Element>
-    if (std::is_trivial<Element>::value) {
+    if constexpr (std::is_trivially_copyable<Element>::value ||
```

Comment:
Same thing here for C++ 17 and the usage of `if constexpr`.

I checked the minimum abseil version, right now it's Abseil 20230125.3 and newer (https://github.com/protocolbuffers/protobuf/blob/07a81cb2d26c21275313b13b6158a34f6fc8ae78/src/google/protobuf/port_def.inc#L134). This version of Abseil already had absl::is_trivially_relocatable: https://github.com/abseil/abseil-cpp/blob/lts_2023_01_25/absl/meta/type_traits.h#L838